bidwire/scrapers/boston_bid_scraper.py

```diff
@@ -1,12 +1,12 @@
 import logging
```

Comment:
Please revert all changes to this file, since it is no longer relevant to your PR. I'm happy to look at clean-up and refactoring of this in a separate PR.